ada/testsuite/tests/contrib/laldoc/basic/test.out

```diff
@@ -0,0 +1,138 @@
+Generating rst
+==============
+
+Unknown annotation: unknown-annotation
+default entity handling for /home/amiard/libadalang/src/libadalang/tmp/contrib__laldoc__basic/base.ads:<SingleTaskDecl ["T"] base.ads:35:4-35:11>
```

Comment:
Not sure this should be part of the baseline :smile:

Reply:
You're right ... I should make the paths relative somehow